src/Network/Network/Common/NetworkResourceManagerProfile.cs

```diff
@@ -1002,6 +1002,9 @@ private static void Initialize()
 
                 cfg.CreateMap<CNM.PSAvailablePrivateEndpointType, MNM.AvailablePrivateEndpointType>();
                 cfg.CreateMap<MNM.AvailablePrivateEndpointType, CNM.PSAvailablePrivateEndpointType>();
+
+                cfg.CreateMap<CNM.PSAutoApprovedPrivateLinkService, MNM.AutoApprovedPrivateLinkService>();
```

Comment:
@bowen0106 is this `AutoApprovedPrivateLinkService` type exposed in a new version of the Network SDK? The build is currently failing because it can't find this type.

Reply:
Yes, it's a new type exposed in new version of SDK. The corresponding Swagger PR is pending.